react/react1/week3/react1-week3/src/components/TodoListRow.jsx

```diff
@@ -0,0 +1,38 @@
+import {useState} from "react"
+import Border from "../Border"
+
+const TodoListRow =(props)=>{
+    const { todo, deleteItem, id } = props
+    const [checked, setChecked]=useState(false)
+    const [editing, setEditing]=useState(false) 
+    const [descriptionInputValue, setDescriptionInputValue] = useState(todo.description)
+
+
+    const updateEditButton =() =>{
+        if (editing === true ){
+            console.log(descriptionInputValue)
```

Comment:
remove console.logs 🧹

Reply:
thank you for reminder I forgot to remove those